inc/mqtthandler.class.php

```diff
@@ -188,6 +188,11 @@ protected function updateInventory($topic, $message) {
       }
    }
 
+   /**
+    * Updates the last contact of the agent
```

Comment:
the data to update here is a datetime, we might tell this in the description of the method.